### PR TITLE
Split feature fully functional from trips page - updates payments owed from all users each time new item is added

### PR DESCRIPTION
**Pull request by austinbrown-opspark**
Split feature available from trips page via "purchases". Buttons for purchases and itinerary are still overlapping - will fix when CSS is fully implemented

### CODE REVIEW

**Review by jldaggs88 — APPROVED** [review]

**Review by shams-ali — CHANGES_REQUESTED** [review]
Travis is failing, fix travis first

File: client/components/Purchases.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import PurchasesForm from './PurchasesForm';
+import PurchasesList from './PurchasesList';
+import axios from 'axios';
+
+const Purchases = () => {
+  const [purchases, setPurchases] = useState([]);
+  useEffect(() => {
+    axios.get('/split')
+      .then(({ data }) => {
+        console.log('resonse from server', data)
```
Comment (jstrzesz) [inline client/components/Purchases.js]: change to console.info

File: client/components/UserTrips.js [hunk 2]
```diff
@@ -8,46 +8,60 @@ import Button from '@material-ui/core/Button';
...
-            <ListItem>
-              Destination:
-              <ListItemText>{ data.tripDestination}</ListItemText>
-            </ListItem>
-          </List>
-        ))}
-        ;
-      </div>
-    );
+  // const render = () => {
```
Comment (jstrzesz) [inline client/components/UserTrips.js:16]: remove comments

**Review by gjones2019 — DISMISSED** [review]

File: client/components/Photos.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        component="label"
+      >
+        Upload File
+        <input
+          type="file"
+          style={{ display: 'none' }}
+          onChange={fileSelectHandler}
+        />
+      </Button >
+      {/* {selected}
```
Comment (jstrzesz) [inline client/components/Photos.js:51]: why is this commented out? If it's for future add a TODO: tag, otherwise remove this commented out code.

Comment (austinbrown-opspark) [reply client/components/Photos.js]: I removed it and pushed changes

**Review by joshjnunez — APPROVED** [review]

**Review by gjones2019 — APPROVED** [review]

**Pull request merged**